proposal-template.abi.md

```diff
@@ -1,31 +1,34 @@
 # Types
```

Comment:
Should this file get renamed to something else now that it is no longer a template?  `wasi-threads.abi.md` seems like the most logical, but I'm not sure that the recommended convention is here.

Reply:
Yes based on wasi-filesystem it looks like `wasi-threads.abi.md` and `wasi-threads.wit.md` are the right names.

Reply:
Good point, renamed to `wasi-threads.*`